BankSDK/GiniBankSDKExample/GiniBankSDKExample/AppCoordinator.swift

```diff
@@ -160,7 +161,37 @@ final class AppCoordinator: Coordinator {
                         for: [GiniCapturePage(document: document, error: nil)])
                 } catch  {
                     self.rootViewController.showErrorDialog(for: error, positiveAction: nil)
-                }
+                }            // When a document is imported with "Open with", a dialog allowing to choose between both APIs
+                // is shown in the main screen. Therefore it needs to go to the main screen if it is not there yet.
+
+            }
+        }
+    }
+    
+    func processExternalDocumentFromPhotos(withUrl url: URL, sourceApplication: String?) {
+        let captureConfiguration = configuration.captureConfiguration()
+        // 1. Build the document
+        let documentBuilder = GiniCaptureDocumentBuilder(documentSource: .appName(name: sourceApplication))
+        documentBuilder.importMethod = .openWith
+        
+        // Get saved imageData which was save in the extension from the group extension UserDefaults
+        // Build document with imageData because it's not possible to do it for the url from Photos
+        if let userDefaults = UserDefaults(suiteName: groupName) {
+            if let data = userDefaults.value(forKey: imageDataKey) {
+             if let doc = documentBuilder.build(with: data as! Data, fileName: "image") {
+                     // When a document is imported with "Open with", a dialog allowing to choose between both APIs
+                     // is shown in the main screen. Therefore it needs to go to the main screen if it is not there yet.
+
+                     self.popToRootViewControllerIfNeeded()
+                     do {
+                         try GiniCapture.validate(doc,
+                                                  withConfig: captureConfiguration)
+                         self.showOpenWithSwitchDialog(
+                             for: [GiniCapturePage(document: doc, error: nil)])
```

Comment:
[Suggestion] since `error` parameter has already a default value, we can remove it from here. Also we can get rid of self here, above and below.
```  self.popToRootViewControllerIfNeeded(),  self.showOpenWithSwitchDialog, self.rootViewController.showErrorDialog```